src/network/networkd-dhcp-static-lease.h

```diff
@@ -0,0 +1,38 @@
+/* SPDX-License-Identifier: LGPL-2.1+ */
+#pragma once
+
+#include <inttypes.h>
+#include <stdbool.h>
+
+#include "conf-parser.h"
+#include "dhcp-server-internal.h"
+
+#include "networkd-network.h"
+#include "networkd-util.h"
+
+
+typedef struct DHCPStaticLease DHCPStaticLease;
+typedef struct Network Network;
+typedef struct NetworkConfigSection NetworkConfigSection;
+
+struct DHCPStaticLease {
+        Network *network;
+        NetworkConfigSection *section;
+
+        be32_t address;
+        DHCPClientId client_id;
```

Comment:
Please store MAC address with `struct ether_addr`. `DHCPClientId` is an internal object of libsystemd-network.